schedule_lessons/dashboard/static/dashboard/css/agenda.css

```diff
@@ -29,11 +29,11 @@ html body {
 }
 
 #noti-triangle-up, #pro-triangle-up {
-	width: 0;
+  width: 0;
   height: 0;
   float: right;
-	border-left: 9px solid transparent;
-	border-right: 9px solid transparent;
+  border-left: 9px solid transparent;
+  border-right: 9px solid transparent;
```

Comment:
these changes look good - please make the necessary changes to the similar code present in other files as well 👍